core/ir/x86/encode.c

```diff
@@ -2835,7 +2838,7 @@ instr_encode_arch(dcontext_t *dcontext, instr_t *instr, byte *copy_pc, byte *fin
             info->opcode);
         info = get_next_instr_info(info);
         /* stop when hit end of list or when hit extra operand tables (OP_CONTD) */
-        if (info == NULL || info->type == OP_CONTD) {
```

Comment:
Why was OP_CONTD removed?

Reply:
There are now OP_CONTD entries that are not END_LIST.

Reply:
OK.